integration/plugin/plugin_test.go

```diff
@@ -0,0 +1,60 @@
+package plugin_test
+
+import (
+	"testing"
+
+	"github.com/ignite/cli/ignite/pkg/cmdrunner/step"
+	envtest "github.com/ignite/cli/integration"
+)
+
+func TestAddPlugin(t *testing.T) {
+	var (
+		env = envtest.New(t)
+		app = env.Scaffold("github.com/test/blog")
+	)
+
+	// TODO use network plugin once finalized
+
+	env.Must(env.Exec("add network plugin locally",
+		step.NewSteps(step.New(
+			step.Exec(envtest.IgniteApp, "plugin", "add", "github.com/aljo242/test-plugin"),
```

Comment:
Maybe we should consider having an official repo with an example plugin that could then be used in the integration tests, for example `github.com/ignite/example-plugin`. Otherwise we might want to use a local path.

Reply:
Yeah, I agree.  I can set that up.

Reply:
The reason why `github.com/aljo242/test-plugin` is not working is because the plugin name declared in the `pluginMap` [here](https://github.com/aljo242/test-plugin/blob/master/main.go#L109) is different that the plugin binary name `test-plugin`.